deployment/ccip/changeset/cs_ccip_home.go

```diff
@@ -66,6 +67,44 @@ func findTokenInfo(tokens []tokenInfo, address common.Address) (string, uint8, e
 	return "", 0, fmt.Errorf("token %s not found in available tokens", address)
 }
 
+func validateExecOffchainConfig(c *pluginconfig.ExecuteOffchainConfig, selector uint64, state CCIPOnChainState) error {
+	if err := c.Validate(); err != nil {
+		return fmt.Errorf("invalid execute off-chain config: %w", err)
+	}
+	// get offRamp
+	if err := state.ValidateRamp(selector, OffRamp); err != nil {
+		return fmt.Errorf("validate offRamp: %w", err)
+	}
+	// TODO how can it be validated for solana?
+	family, err := chain_selectors.GetSelectorFamily(selector)
+	if err != nil {
+		return err
+	}
+	if family == chain_selectors.FamilyEVM {
+		offRamp := state.Chains[selector].OffRamp
+		// get permissionlessExecutionThresholdSeconds
+		dCfg, err := offRamp.GetDynamicConfig(nil)
```

Comment:
nit: pass in `ctx` to this validator? Should be available from the environment I think